File: app/src/androidTest/java/ru/ltst/u2020mvp/TestU2020Application.java
package ru.ltst.u2020mvp;


public class TestU2020Application extends U2020App {
    private TestU2020Component component;

    @Override
    public void buildComponentAndInject() {
        component = DaggerTestU2020Component.builder()
                .u2020AppModule(new U2020AppModule(this))
                .build();
        component.inject(this);
    }

    @Override
    public U2020Component component() {
        return component;
    }

    public TestU2020Component getTestComponent() {
        return component;
    }
}

File: app/src/androidTest/java/ru/ltst/u2020mvp/TestU2020Application.kt
package ru.ltst.u2020mvp


class TestU2020Application : U2020App() {
    lateinit var testComponent: TestU2020Component
        private set

    override fun buildComponentAndInject() {
        testComponent = DaggerTestU2020Component.builder()
                .u2020AppModule(U2020AppModule(this))
                .build()
        testComponent.inject(this)
    }

    override fun component(): U2020Component {
        return testComponent
    }
}
